sql-plugins/src/main/java/io/cdap/pipeline/sql/plugins/BigQuerySQLSink.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.pipeline.sql.plugins;
+
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.annotation.Plugin;
+import io.cdap.pipeline.sql.api.core.Column;
+import io.cdap.pipeline.sql.api.core.StructuredQuery;
+import io.cdap.pipeline.sql.api.core.Table;
+import io.cdap.pipeline.sql.api.template.SQLSink;
+
+import java.util.regex.Pattern;
+
+/**
+ * A BigQuery SQL sink.
+ */
+@Plugin(type = SQLSink.PLUGIN_TYPE)
+@Name("BigQueryTable")
+@Description("A BigQuery sink.")
+public class BigQuerySQLSink extends SQLSink {
+  private final BigQuerySQLSinkConfig config;
+
+  public BigQuerySQLSink(BigQuerySQLSinkConfig config) {
+    this.config = config;
+  }
+
+  @Override
+  public StructuredQuery constructQuery() {
+    Column allColumns = new Column("*", null, null, null);
```

Comment:
its kind of annoying to pass in a bunch of nulls, let's add a builder to Column so that this isn't needed

Reply:
Sounds good, added builder pattern. This is especially true since we may end up adding aggregation later too.